modules/liveIntentIdSystem.js

```diff
@@ -107,6 +107,12 @@ function initializeLiveConnect(configParams) {
   }
 
   const liveConnectConfig = parseLiveIntentCollectorConfig(configParams.liCollectConfig);
+  
+  if (!liveConnectConfig.appId && configParams.distributorId) {
+    liveConnectConfig.distributorId = configParams.distributorId;
```

Comment:
LiveConnect already ignores the `distributorId` when an `appId` is present. Duplicating this check here to have all distributorId related config decisions in one place.

Reply:
since we have two different places where the publisher id and distributor id (publisher id ends up in a separate column whereas distributor id is going be a part of destination field), I would recommend we keep both. Happy to learn more about what you think.

Reply:
@SuperIzya You mean in one place like here or in one place in live-connect?

Reply:
IMHO, it would make more sense to have in live-connect. But I'm fine either way, as long as it's the **only** place